src/plugins/vis_type_timeseries/public/application/components/lib/convert_series_to_vars.js

```diff
@@ -54,6 +54,14 @@ export const convertSeriesToVars = (series, model, dateFormat = 'lll', getConfig
         };
         set(variables, varName, data);
         set(variables, `${_.snakeCase(row.label)}.label`, row.label);
+
+        if (row.labelFormatted) {
+          set(
+            variables,
+            `${_.snakeCase(row.label)}.formatted`,
```

Comment:
I was thinking of it more like `label.formatted` but maybe this is my personal opinion.